dask/dataframe/tests/test_dataframe.py

```diff
@@ -3575,10 +3581,12 @@ def return_df2(x):
     # Series to completely different DataFrame
     with warnings.catch_warnings():
         warnings.simplefilter("ignore", UserWarning)
-        result = ddf.x.apply(return_df2)
+        with pytest.warns(FutureWarning, match="Returning a DataFrame"):
```

Comment:
Want to ignore the warnings here for now. A couple of cases aren't raising warnings yet (which seems buggy).